src/controllTools.js

```diff
@@ -0,0 +1,67 @@
+import { checkbox, editTask } from './index.js';
```

Comment:
Great job so far, in my opinion there is still something that needs improvement, , you can see the video below:
https://www.loom.com/share/4a456d42e49340c0b24085b944874b2d